src/IssueViz.Security/HotspotsList/TableDataSource/HotspotsTableColumns.cs

```diff
@@ -29,7 +29,8 @@ internal static class HotspotsTableColumns
         public static IReadOnlyList<ColumnState> InitialStates { get; } = new[]
         {
             new ColumnState(StandardTableColumnDefinitions.ErrorCode, true, 50),
-            new ColumnState(StandardTableColumnDefinitions.Text, true, 450),
+            new ColumnState(StandardTableColumnDefinitions.Priority, true, 50),
+            new ColumnState(StandardTableColumnDefinitions.Text, true, 300),
```

Comment:
Misc: the toolwindow width is 700, so I matched the column width. I don't know if that makes any difference.